packages/ibm-products/src/components/BigNumbers/BigNumbers.test.js

```diff
@@ -12,36 +12,25 @@ import { Edit } from '@carbon/react/icons';
 import { pkg, carbon } from '../../settings';
 import uuidv4 from '../../global/js/utils/uuidv4';
 import { BigNumbers } from '.';
-import { BigNumbersSize } from './constants';
+import { BigNumbersSkeleton } from './BigNumbersSkeleton';
+import { getSupportedLocale } from './constants';
 
 const blockClass = `${pkg.prefix}--big-numbers`;
 const skeletonBlockClass = `${pkg.prefix}--big-numbers-skeleton`;
 const componentName = BigNumbers.displayName;
+const componentNameSkeleton = BigNumbersSkeleton.displayName;
 
 // values to use
 const iconButtonClassName = `iconButtonClass-${uuidv4()}`;
 const className = `class-${uuidv4()}`;
 const dataTestId = uuidv4();
 
 const renderBigNumbers = ({ ...rest } = {}) =>
-  render(
-    <BigNumbers
-      label="Label"
-      value={12345}
-      total={55555}
-      percentage={false}
-      size={BigNumbersSize.Default}
-      forceShowTotal={false}
-      trending={false}
-      truncate={true}
-      locale="en-US"
```

Comment:
Removing these "default" values means each test below only has to pass the props specific to the test, instead of having to consider "add/change/remove" every time.